llvm/tools/llvm-cov/CodeCoverage.cpp

```diff
@@ -774,6 +775,11 @@ int CodeCoverageTool::run(Command Cmd, int argc, const char **argv) {
       "check-binary-ids", cl::desc("Fail if an object couldn't be found for a "
                                    "binary ID in the profile"));
 
+  cl::opt<std::string> DebugInfoFilename(
+      "debug-info", cl::init(""),
```

Comment:
```suggestion
      "debug-info",
```
NIT: The default value is already the empty string